org-formation/organization.yaml

```diff
@@ -138,6 +138,7 @@ Organization:
         - !Ref RestrictDnsRegistrationSCP
       Accounts:
         - !Ref BuA2aDwAccount
+        - !Ref BWmErzkAccount
```

Comment:
is there a reason to use a random string (BWmErzk) rather than the user's name when creating a personal account?

Reply:
in case an employee leaves we can assign the account to someone else by updating the AccountOwner tag.